network/iptables.go

```diff
@@ -11,6 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//go:build !windows
```

Comment:
As we already have line 15, does this line bring any value?

Reply:
Remove the useless `go:build` line which added by goland automatic.